impeller/renderer/backend/vulkan/test/mock_vulkan.cc

```diff
@@ -52,6 +60,8 @@ class MockDevice final {
   Mutex command_buffers_mutex_;
   std::vector<std::unique_ptr<MockCommandBuffer>> command_buffers_
       IPLR_GUARDED_BY(command_buffers_mutex_);
+
+  std::function<std::shared_ptr<MockFence>()> fence_factory_;
```

Comment:
I think this is unused.

Reply:
Ah good catch, artifact of old merge. Removed!

(I wish we had private code unused warnings)